New project fails to build

**I'm submitting a bug report**

* **Library Version:**
v1.0.0-beta.12

**Please tell us about your environment:**
* **Operating System:**
OSX 10.14

* **Node Version:**
10.15.1

* **NPM Version:**
6.4.1

* **Browser:**
all (build issue)

* **Language:**
ESNext

* **Loader/bundler:**
Webpack

**Current behavior:**
Installed Aurelia CLI, created blank project accepting all defaults - no modifications made myself following this
Running `au build` or `au run --watch` fails with errors:

```
ERROR in ./node_modules/@babel/polyfill/lib/index.js
Module not found: Error: Can't resolve 'core-js/fn/promise/finally' in '<my_dir>/aurelia-app/node_modules/@babel/polyfill/lib'
 @ ./node_modules/@babel/polyfill/lib/index.js 19:0-37
 @ ./src/main.js
 @ ./node_modules/aurelia-webpack-plugin/runtime/empty-entry.js
 @ multi aurelia-webpack-plugin/runtime/empty-entry aurelia-webpack-plugin/runtime/pal-loader-entry webpack-dev-server/client aurelia-webpack-plugin/runtime/empty-entry aurelia-webpack-plugin/runtime/pal-loader-entry aurelia-bootstrapper

ERROR in ./node_modules/@babel/polyfill/lib/index.js
Module not found: Error: Can't resolve 'core-js/fn/string/pad-end' in '<my_dir>/aurelia-app/node_modules/@babel/polyfill/lib'
 @ ./node_modules/@babel/polyfill/lib/index.js 9:0-36
 @ ./src/main.js
 @ ./node_modules/aurelia-webpack-plugin/runtime/empty-entry.js
 @ multi aurelia-webpack-plugin/runtime/empty-entry aurelia-webpack-plugin/runtime/pal-loader-entry webpack-dev-server/client aurelia-webpack-plugin/runtime/empty-entry aurelia-webpack-plugin/runtime/pal-loader-entry aurelia-bootstrapper

ERROR in ./node_modules/@babel/polyfill/lib/index.js
Module not found: Error: Can't resolve 'core-js/fn/string/pad-start' in '<my_dir>/aurelia-app/node_modules/@babel/polyfill/lib'
 @ ./node_modules/@babel/polyfill/lib/index.js 7:0-38
 @ ./src/main.js
 @ ./node_modules/aurelia-webpack-plugin/runtime/empty-entry.js
 @ multi aurelia-webpack-plugin/runtime/empty-entry aurelia-webpack-plugin/runtime/pal-loader-entry webpack-dev-server/client aurelia-webpack-plugin/runtime/empty-entry aurelia-webpack-plugin/runtime/pal-loader-entry aurelia-bootstrapper

ERROR in ./node_modules/@babel/polyfill/lib/index.js
Module not found: Error: Can't resolve 'core-js/fn/symbol/async-iterator' in '<my_dir>/aurelia-app/node_modules/@babel/polyfill/lib'
 @ ./node_modules/@babel/polyfill/lib/index.js 11:0-43
 @ ./src/main.js
 @ ./node_modules/aurelia-webpack-plugin/runtime/empty-entry.js
 @ multi aurelia-webpack-plugin/runtime/empty-entry aurelia-webpack-plugin/runtime/pal-loader-entry webpack-dev-server/client aurelia-webpack-plugin/runtime/empty-entry aurelia-webpack-plugin/runtime/pal-loader-entry aurelia-bootstrapper
Child html-webpack-plugin for "index.html":
         Asset      Size  Chunks  Chunk Names
    index.html  1.36 MiB       0
    Entrypoint undefined = index.html
    [8XHo] ./node_modules/html-webpack-plugin/lib/loader.js!./index.ejs 627 bytes {0} [built]
    [LvDl] ./node_modules/lodash/lodash.js 527 KiB {0} [built]
    [YuTi] (webpack)/buildin/module.js 497 bytes {0} [built]
    [yLpj] (webpack)/buildin/global.js 472 bytes {0} [built]
ℹ ｢wdm｣: Failed to compile.
```

* **What is the expected behavior?**
The project should build and run correctly

--- a/lib/resources/content/webpack.config.template.js
+++ b/lib/resources/content/webpack.config.template.js
@@ -38,7 +38,7 @@ module.exports = ({ production, server, extractCss, coverage, analyze, karma } =
     // @if transpiler.id='babel'
     extensions: ['.js'],
     // @endif
-    modules: [srcDir, nodeModulesDir],
+    modules: [srcDir, 'node_modules'],
     // Enforce single aurelia-binding, to avoid v1/v2 duplication due to
     // out-of-date dependencies on 3rd party aurelia plugins
     alias: { 'aurelia-binding': path.resolve(__dirname, 'node_modules/aurelia-binding') }

